File: backend/src/main/java/edu/netcracker/backend/model/Service.java
package edu.netcracker.backend.model;

import edu.netcracker.backend.dao.annotations.Attribute;
import edu.netcracker.backend.dao.annotations.PrimaryKey;
import edu.netcracker.backend.dao.annotations.Table;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@EqualsAndHashCode(onlyExplicitlyIncluded = true)
@Table("service")
public class Service {

    @PrimaryKey("service_id")
    @EqualsAndHashCode.Include
    private Long serviceId;

    @Attribute("service_name")
    private String serviceName;

    @Attribute("service_description")
    private String serviceDescription;

    private Integer servicePrice;
}

File: frontend/src/main/web/src/app/pages/admin/bundles/shared/model/service.ts
export class Service {
  constructor(
    public service_id : number,
    public service_name : string,
    public service_price : number,
    public item_number : number,
  ) {}

  getTotalPrice() {
    return ((this.service_price || 0) * (this.item_number || 0));
  }
}
